packages/drivers/odsp-driver/src/test/mockFetch.ts

```diff
@@ -4,7 +4,7 @@
 */
 
 /* eslint-disable @typescript-eslint/ban-types */
-
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Seems like we should be able to solve this with tests using an extended eslint config? Then we can allow with `devDependencies: true` for `import/no-extraneous-dependencies`.

Reply:
I am doing it globally with the change in the `eslint-config-fluid`.  I just need to check that in, publish it before I can consume it.  So this line will be removed then.